builder/src/lib.rs

```diff
@@ -51,18 +51,21 @@ fn run_cmd(cmd: &mut Command) -> io::Result<()> {
     }
 }
 
-fn run_cmd_stdout(cmd: &mut Command, input: Option<&[u8]>) -> io::Result<String> {
+pub fn run_cmd_stdout(cmd: &mut Command, input: Option<&[u8]>) -> io::Result<String> {
     cmd.stdin(Stdio::piped());
     cmd.stdout(Stdio::piped());
 
     let mut child = cmd.spawn()?;
-    if let (Some(mut stdin), Some(input)) = (child.stdin.take(), input) {
-        std::io::Write::write_all(&mut stdin, input)?;
+    if let (Some(stdin), Some(input)) = (child.stdin.take(), input) {
+        let mut writer = BufWriter::new(stdin);
```

Comment:
Does this change actually do anything?

Reply:
No. It was a left-over attempt to fix the broke pipe.